src/main/java/org/manifold/compiler/front/ConstantValueVertex.java

```diff
@@ -0,0 +1,56 @@
+package org.manifold.compiler.front;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+import org.manifold.compiler.TypeValue;
+import org.manifold.compiler.Value;
+
+public class ConstantValueVertex extends ExpressionVertex {
```

Comment:
I guess my problem is just that this `ValueVertex` class looks *suspiciously* similar to the `Value` class